testing-utils/src/lib/fixtures/configs/core-config.mock.ts

```diff
@@ -1,7 +1,5 @@
-import { CoreConfig } from '../../../../../packages/models/src';
-
 export const CORE_CONFIG_MOCK = {
-  persist: { outputDir: 'test', filename: 'output.json' },
+  persist: { outputDir: 'test' },
```

Comment:
must be a mistake as:
- should be `report`. `output` is used for runner output.
- should be by default `report´